$NetBSD: patch-ab,v 1.9 2000/07/12 20:07:50 jlam Exp $

Use -fPIC on NetBSD
On Solaris, fix up the tcl library name, so it's the same as NetBSD
for pkgsrc internal use (PLIST entries)
On Solaris, do not use dots in version names for the library (same
reason as previous)
Use -fPIC on Solaris too, otherwise we overflow a 2048 entry table.
Set TCL_SHARED_BUILD to 1.

--- configure.orig	Tue Mar  9 06:51:26 1999
+++ configure	Thu Apr  1 14:12:15 1999
@@ -4096,7 +4096,7 @@
 if eval "test \"`echo '$ac_cv_header_'$ac_safe`\" = yes"; then
   echo "$ac_t""yes" 1>&6
   
-	    SHLIB_CFLAGS="-fpic"
+	    SHLIB_CFLAGS="-fPIC"
 	    SHLIB_LD="ld -Bshareable -x"
 	    SHLIB_LD_LIBS=""
 	    SHLIB_SUFFIX=".so"
@@ -4224,7 +4224,7 @@
 	TCL_LIB_VERSIONS_OK=nodots
 	;;
     SunOS-5*)
-	SHLIB_CFLAGS="-KPIC"
+	SHLIB_CFLAGS="-fPIC"
 	SHLIB_LD="/usr/ccs/bin/ld -G -z text"
 
 	# Note: need the LIBS below, otherwise Tk won't find Tcl's
@@ -4236,6 +4236,9 @@
 	DL_LIBS="-ldl"
 	LD_FLAGS=""
 	LD_SEARCH_FLAGS='-R ${LIB_RUNTIME_DIR}'
+	    TCL_SHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.so.1.0'
+	    TCL_UNSHARED_LIB_SUFFIX='${TCL_TRIM_DOTS}\$\{DBGX\}.a'
+	TCL_LIB_VERSIONS_OK=nodots
 	;;
     ULTRIX-4.*)
 	SHLIB_CFLAGS="-G 0"
@@ -4679,6 +4682,8 @@
     eval "TCL_LIB_FILE=libtcl${TCL_UNSHARED_LIB_SUFFIX}"
     MAKE_LIB="ar cr \${TCL_LIB_FILE} \${OBJS}"
 fi
+
+TCL_SHARED_BUILD=1
 
 # Note:  in the following variable, it's important to use the absolute
 # path name of the Tcl directory rather than "..":  this is because
